applicable_pose_generator/include/applicable_pose_generator/pose_reachability_filter.h

```diff
@@ -0,0 +1,54 @@
+/*
+    Software License Agreement (Apache License)
+    Copyright (c) 2014, Southwest Research Institute
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+#ifndef POSE_REACHABILITY_FILTER_H_
+#define POSE_REACHABILITY_FILTER_H_
+
+
```

Comment:
In every .cpp, .c .hpp or .h file, organize all your #includes 
only have additional empty lines between groups of similar includes